internal/cli/args.go

```diff
@@ -48,44 +54,86 @@ tilt args -- --foo=bar frontend backend
 	return cmd
 }
 
-type httpPoster func(url string, contentType string, body io.Reader) (*http.Response, error)
+func newClient(ctx context.Context) (client.Client, error) {
+	getter, err := wireClientGetter(ctx)
+	if err != nil {
+		return nil, err
+	}
 
-func (c *argsCmd) run(ctx context.Context, args []string) error {
-	// require --clear instead of an empty args list to ensure an experimental `tilt flags` doesn't unintentionally wipe state
-	if len(args) == 0 {
-		if !c.clear {
-			return errors.New("no args specified. If your intent is to empty the args, run `tilt args --clear`.")
+	cfg, err := getter.ToRESTConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	ctrlclient, err := client.New(cfg, client.Options{Scheme: v1alpha1.NewScheme()})
+	if err != nil {
+		return nil, err
+	}
+
+	return ctrlclient, err
+}
+
+func parseEditResult(b []byte) ([]string, error) {
+	lines := strings.Split(string(b), "\n")
```

Comment:
Might be worth swapping to a `bufio.Scanner` here - I'm guessing Windows editors will be `\r\n` line-terminated?

Reply:
oy, I guess I should figure out a windows test